src/components/header/burgerMenu.tsx

```diff
@@ -19,15 +21,20 @@ export const BurgerMenu = ({
 }) => {
   const [isOpen, setIsOpen] = useState(false);
   const [query, setQuery] = useState("");
+  const [searchList, setSearchList]: [any, any] = useState([]);
```

Comment:
This don't work? `const [searchList, setSearchList] = useState<(IProject | INews)[] | false>([])`